README.rst

```diff
@@ -21,7 +21,17 @@ To install the latest release of scikit-bio::
 
     pip install scikit-bio
 
-Equivalently, you can use the ``conda`` package manager available in `Anaconda <http://continuum.io/downloads>`_ or `miniconda <http://conda.pydata.org/miniconda.html>`_ to install scikit-bio and its dependencies without having to compile them::
+Some plotting features require extra dependencies, and an
+``ImportError`` will be raised if you try to use them without them.
```

Comment:
Good catch, using `them` with two meanings one word apart is horrible style, I'll fix it. Thanks!